packages/stateless/components/inputs/SelectInput.tsx

```diff
@@ -41,27 +47,41 @@ export const SelectInput = <
   )
 
   return (
-    <select
-      className={clsx(
-        'secondary-text rounded-md bg-transparent py-3 px-4 text-text-body ring-1 transition placeholder:text-text-tertiary focus:outline-none focus:ring-2',
-        error
-          ? 'ring-border-interactive-error'
-          : 'ring-border-primary focus:ring-border-interactive-focus',
-        props.disabled && 'appearance-none',
-        className
+    <div className={clsx('relative', containerClassName)}>
```

Comment:
**thought**: my laziness is kicking in and i dont want to go through each class here lol, but a quick check of this and it looks good!